MDANSE/Src/MDANSE/Framework/Configurators/WeightsConfigurator.py

```diff
@@ -109,20 +109,24 @@ def configure(self, value):
         self.error_status = "OK"
 
     def get_weights(self):
-        ascfg = self._configurable[self._dependencies["atom_selection"]]
+        atom_selection_configurator = self._configurable[
+            self._dependencies["atom_selection"]
+        ]
 
         weights = {}
-        for i in range(ascfg["selection_length"]):
-            name = ascfg["names"][i]
-            for el in ascfg["elements"][i]:
-                p = self._trajectory.get_atom_property(el, self["property"])
+        for i in range(atom_selection_configurator["selection_length"]):
+            name = atom_selection_configurator["names"][i]
+            for element in atom_selection_configurator["elements"][i]:
```

Comment:
Bit verbose due to length of names, but maybe clearer?
```suggestion
        for name, elements in itertools.islice(
            zip(atom_selection_configurator["names"],
                atom_selection_configurator["elements"]),
            atom_selection_configurator["selection_length"]
        ):
            for element in elements:
```